falcon/request.py

```diff
@@ -188,6 +189,9 @@ class Request(object):
             A dict of name/value cookie pairs.
             See also: :ref:`Getting Cookies <getting-cookies>`
 
+        json (dict): Attempts to parse the request stream as a Json string.
```

Comment:
I think it should be called "JSON".<sup>[[1]](http://www.w3.org/TR/html-json-forms/)</sup>